### PR TITLE
Added new team members

### PR DESCRIPTION
**Pull request by hikes101**
- added new team members
- removed last names
- added placeholder image for missing photos (so far the free version with watermark until I get the OK from Bianca)

image

### CODE REVIEW

**Comment by hikes101** [comment]
[Menschen: Add Akash](https://trello.com/c/2oHKnYA7/169-menschen-add-akash)

**Comment by hikes101** [comment]
[Menschen: Add Fabian, Sabine, Caroline](https://trello.com/c/ZCcdTYR1/170-menschen-add-fabian-sabine-caroline)

File: data/team.json [hunk 1]
```diff
@@ -1,58 +1,88 @@
...
+      "description": "Fabian hat an der Technischen Uni Berlin Physik mit dem Schwerpunkt theoretische Physik studiert und sich in diesem Rahmen sowohl viel mit Mathe auseinandergesetzt als auch wissenschaftliches Programmieren gelernt. Für Die Lernwerkstatt ist er in der Lernbegleitung tätig und unterstützt junge Menschen ihren Interessen zu folgen. <br />Fabian zu seiner Motivation, bei uns mitzumachen: „Auf der Ausbruch!-Messe 2018 in Berlin habe ich die Lernwerkstatt und das Freilernen kennengelernt. Wie die meisten etablierten Systeme in unserer Gesellschaft, nehme ich auch das momentane Bildungswesen als etwas sehr gewaltsames wahr, in dessen Konsequenz sowohl die Lehrer*innen als auch die Schüler*innen frustriert und überfordert sind, weil sie nicht ihren intrinsischen Interessen folgen können. Ich möchte deshalb mit der Lernwerkstatt einen anderen liebevollen Weg zeigen und gehen, auf dem Bildung und Lernen nicht mit Leistungsdruck, erzwungener Disziplin und starren Strukturen assoziiert wird, sondern Lernen als ein lebendiger interessengetriebener Wachstumsprozess, der uns allen eigen ist, mit Freude und Selbstbestimmung praktiziert wird.“"
```
Comment (pheeria) [inline data/team.json]: Did you try to use `\n` instead of `<br />`? :)

File: src/views/partials/team/member.hbs [hunk 2]
```diff
@@ -18,7 +18,7 @@
             <img src="{{img}}" alt="{{name}}" style="width: 100%; height: auto; max-width: 160px" />
           </div>
           <div class="col-sm-9 text-left">
-            {{description}}
+            {{{description}}}
```
Comment (pheeria) [inline src/views/partials/team/member.hbs]: If we get rid of `<br>` we could (and probably should) store the data in plain text, not html.

File: data/team.json [hunk 3]
```diff
@@ -1,58 +1,88 @@
...
+      "description": "André unterrichtet seit 2009 hauptsächlich Mathe, Physik und Englisch. Er hat schon Menschen allen Alters unterrichtet und war in verschiedenen Ländern tätig. Lernen ist seine große Leidenschaft. Er interessiert sich für beinahe alles und glaubt fest daran, dass mit der richtigen Einstellung und den richtigen Lernmethoden nichts unmöglich ist. <br />Die Lernwerkstatt ist für ihn eine Chance, seine Freude am Lernen und am Ausprobieren mit anderen Menschen zu teilen. Er wünscht sich einen Ort, an dem eine aktive Lerngemeinde entsteht und an dem offen und ehrlich miteinander umgegangen wird. André betreut die Lernbegleitung, den Bereich Mathe und Naturwissenschaften, die Projektfinanzierung sowie die allgemeine Projektkoordination."
```
Comment (pheeria) [inline data/team.json]: Let's add surnames in descriptions, like Bianca suggested.

File: src/views/partials/team/member.hbs (hunk 2 above)
Comment (hikes101) [reply src/views/partials/team/member.hbs]: Looks like handlebars doesn't support that. We would have to use another helper function for that.

**Pull request merged**